sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/channel/DistributeInfoTracker.java

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.appcenter.distribute.channel;
+
+import android.support.annotation.NonNull;
+
+import com.microsoft.appcenter.channel.AbstractChannelListener;
+import com.microsoft.appcenter.ingestion.models.Log;
+
+/**
+ * Channel listener which adds extra fields to logs.
+ */
+public class DistributeInfoTracker extends AbstractChannelListener {
+
+    /**
+     * Distribution group ID that is added to logs (if exists).
+     */
+    private String mDistributionGroupId;
+
+    /**
+     * Init.
+     *
+     * @param distributionGroupId distribution group ID that is added to logs (if exists).
+     */
+    public DistributeInfoTracker(String distributionGroupId) {
+        mDistributionGroupId = distributionGroupId;
+    }
+
+    @Override
+    public void onEnqueuingLog(@NonNull Log log, @NonNull String groupName) {
```

Comment:
please add synchronized before public and void

Reply:
Fixed. Thank you.